packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml

```diff
@@ -93,6 +93,25 @@ event:
     data_sensitivity:
       - interaction
 
+  app_framework:
+    type: string
+    description: |
+      The name of the framework used by the app (ie React or Backbone).
+    lifetime: ping
+    send_in_pings:
+      - accounts-events
```

Comment:
I think there's some desire to use the events pings only in the future.  @akkomar isn't that right?